### PR TITLE
fix(version): 'nightly' is no longer a valid alias

### PR DESCRIPTION
**Pull request by lwander**

### CODE REVIEW

File: halyard-config/src/main/java/com/netflix/spinnaker/halyard/config/model/v1/node/DeploymentConfiguration.java [hunk 1]
```diff
@@ -45,7 +45,7 @@
    *
    * @see Halconfig#halyardVersion
    */
-  String version = "nightly";
+  String version = "master-latest-unvalidated";
```
Comment (ewiseblatt) [inline halyard-config/src/main/java/com/netflix/spinnaker/halyard/config/model/v1/node/DeploymentConfiguration.java]: why is this "master"?
doesnt "master" refer to the branch?

Comment (lwander) [inline halyard-config/src/main/java/com/netflix/spinnaker/halyard/config/model/v1/node/DeploymentConfiguration.java]: Yup - it's effectively what "nightly" was before we stopped using that alias.

Comment (ewiseblatt) [inline halyard-config/src/main/java/com/netflix/spinnaker/halyard/config/model/v1/node/DeploymentConfiguration.java]: I mean, how do you know you are in master? This shouldnt be a static string. In fact none of the tag should be. The naming policy should be defined externally.

Comment (lwander) [inline halyard-config/src/main/java/com/netflix/spinnaker/halyard/config/model/v1/node/DeploymentConfiguration.java]: This is a default/placeholder before we get have a proper "latest" alias that points to the latest validated Spinnaker. I know that for the next week before we release, this alias is just a QOL improvement that effectively points at what "nightly" pointed at.

**Pull request merged**